app/commands.py

```diff
@@ -914,3 +914,26 @@ def populate_annual_billing_with_defaults(year, missing_services_only):
 
     for service in active_services:
         set_default_free_allowance_for_service(service, year)
+
+
+@notify_command(name='tmp-backfill-custom-broadcast-areas')
+def tmp_backfill_custom_broadcast_areas():
+    from app import db
+    from app.models import BroadcastMessage
+
+    custom_broadcasts = BroadcastMessage.query \
+        .filter(BroadcastMessage.api_key_id != None) \
```

Comment:
You can put both conditions in the `.filter` (e.g. we do that [here](https://github.com/alphagov/notifications-api/blob/master/app/dao/annual_billing_dao.py#L30)), but as with the other comment it's probably not worth making a change if this command will be removed soon.

Reply:
I personally prefer is way as I find it more readable vs. having both conditions on the same line.